### DISCUSSION TITLE
myStockMaster

### DISCUSSION THREAD

**Discussion by GAYE34** [Zakarialabib/myStockMaster#51] [Q&A]
hello how are you i successfully clone the project on my server but when i try to access it i get an error with the languages table can you help. this is the error code. any idea on how to resolve the issue.
[ Illuminate
 \ 
Database
 \ 
QueryException
PHP 8.0.28
9.52.4
SQLSTATE[HY000] [2002] Permission denied
SELECT
  *
FROM
  information_schema.tables
WHERE
  table_schema = laravel
  AND table_name = languages
  AND table_type = 'BASE TABLE'

**Comment by elmarzouguidev** [comment]
Can you tell us wish database you use Sqlit or mysql or ... ?
or you check this answer here 
https://stackoverflow.com/questions/34675607/apache-php-mysql-pdo-permission-denied-laravel

**Reply by GAYE34** [reply]
am using mysql

**Comment by Zakarialabib** [comment]
I suggest migrating database with seeds
Php artisan migrate:fresh --seed
